_data/master/navbars/networking.yml

```diff
@@ -0,0 +1,56 @@
+bigheader: "Networking"
+path: /networking
+order: 2
+toc:
+- title: Networking
```

Comment:
I think this section is missing the new stuff from v3.5 re: IPAM based on topology.

Reply:
Yes, I had been working to update this PR around that so it's still a little in progress. Now we have v3.5, I'll need to rebase again, as you noted.